co-log-core/src/Colog/Core/IO.hs

```diff
@@ -65,6 +60,43 @@ foo
 withLogStringFile :: MonadIO m => FilePath -> (LogAction m String -> IO r) -> IO r
 withLogStringFile path action = withFile path AppendMode $ action . logStringHandle
 
+----------------------------------------------------------------------------
+-- Show
+----------------------------------------------------------------------------
+
+{- | Action that prints to stdout using 'Show'.
+
+>>> unLogAction logPrint 5
+5
+-}
+logPrint :: forall a m . (Show a, MonadIO m) => LogAction m a
+logPrint = LogAction $ liftIO . print
+
+{- | Action that prints to stderr using 'Show'.
+
+>>> unLogAction logPrintStderr 5
+5
+-}
+logPrintStderr :: forall a m . (Show a, MonadIO m) => LogAction m a
+logPrintStderr = logPrintHandle stderr
+
+{- | Action that prints to 'Handle' using 'Show'.
```

Comment:
Very minor, but looking at the way `Handle` is described in [other docs](https://hackage.haskell.org/package/base-4.8.2.0/docs/GHC-IO-Handle.html), we should probably do the following:
```suggestion
{- | Action that prints to a 'Handle' using 'Show'.
```